src/main/java/com/smartsheet/api/models/DeleteUserParameters.java

```diff
@@ -0,0 +1,115 @@
+package com.smartsheet.api.models;
+
+/*
+ * #[license]
+ * Smartsheet SDK for Java
+ * %%
+ * Copyright (C) 2014 Smartsheet
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * %[license]
+ */
+
+import com.smartsheet.api.internal.util.QueryUtil;
+
+import java.util.HashMap;
+
+public class DeleteUserParameters {
+    /**
+     * Represents the ID of the user to transfer ownership to
+     */
+    private Long transferToId;
+
+    /**
+     * Determines whether or not to transfer sheets
+     */
+    private Boolean transferSheets;
+
+    /**
+     * Determines to remove the user from sharing for all sheets/workspaces in the organization
+     */
+    private Boolean removeFromSharing;
+
+    public DeleteUserParameters() {}
+
+    public DeleteUserParameters(Long transferToId, Boolean transferSheets, Boolean removeFromSharing) {
```

Comment:
Add documentation to this constructor so that it will show up in our auto generated docs as well as the IDE when the user is wondering what the parameters are for.